wikichange/src/content/content.js

```diff
@@ -65,35 +90,36 @@ const renderSlider = async (creationDate) => {
     let totalDaysDiff = (now.getTime() - creationDate.getTime()) / (1000 * 3600 * 24);
     let viewsEditsChart = document.getElementById("viewsEditsChart");
     let sliderDiv = document.createElement("div");
+    sliderDiv.setAttribute("id", "sliderDiv");
     let initialDate = new Date();
     initialDate.setDate(now.getDate() - totalDaysDiff * 0.5);
 
     let curRevisionId = (await fetchRevisionFromDate(title, now))[0];
     let oldRevisionId = (await fetchRevisionFromDate(title, initialDate))[0];
-
+    console.log(fetchChangeWithHTML(curRevisionId, oldRevisionId));
     highlightRevisionBetweenRevisionIds(title, curRevisionId, oldRevisionId);
 
-    sliderDiv.innerHTML = `<div style="direction: rtl">${now.toISOString().slice(0, 10)}  
-                                <input type="range" id="graphSlider" value="50" min="0" max="100" style="width:60%;">  
-                                ${creationDate.toISOString().slice(0, 10)}
+    sliderDiv.innerHTML = `<div style="padding-left:5%; direction: rtl;">  
+                                <input type="range" id="graphSlider" value="50" min="0" max="100" style="width:90%;">  
                             </div>
-                            <br/><input type="date" value="${initialDate
+                            <input type="date" value="${initialDate
                                 .toISOString()
                                 .slice(0, 10)}" id="dateOutput" name="dateOutput" style="text-align: center;"> 
                                 <button id = "highlightButton">Highlight</button> <div id="loader"></div>
                                 <p></p>
-                                <button id = "revisionButton">Go To Revision Page</button></div>
-                                <p id="revisionDate">Showing highlight for closest revision (<b>date: <span id="closesRev">
-                                ${(await fetchRevisionFromDate(title, initialDate))[1].slice(0, 10)}</span></b>)</p>`;
+                                <div class="card" style="max-width: 100%;border-style: solid;">
+                                    <div class="card-body">
+                                    <p class="card-text" id="revisionContext"> Compared to the <a href=${(getRevisionPageLink(curRevisionId, oldRevisionId))} target="_blank">${(await fetchRevisionFromDate(title, initialDate))[1].toLocaleDateString().slice(0, 10)}</a> version, the current page has 30 differences highlight below</p>
```

Comment:
We should change this wording:
1) We need to make sure to explain that the date showing is the closest revision (can be confusing if people expect to be the same as the one in the date picker)
2) Mention that the additions are highlighted in green and no deletions are visible
3) Maybe include a mention to go to the popup to learn more
@shirin601 @sukritsangvong @etiennerichart  any thoughts? We probably need to come up with a blurb together

Reply:
I think the click to go to the revision page is not that clear. I would prefer it to be a button next to the highlight button. Thoughts? @LuisaE @shirin601 @RichartE 

For the content, we can rewrite it to something like
``` 
Comparing the current Wikipedia page to the closest revision of the chosen time (mm/dd/yyyy). Newly added texts are highlighted in green, but the deletions are not included.
```

Shirin can make more edits to this later. Since we want something to be up there first, we can probably remove the 30 diff for now and can add it later when we have something to count the differences.

Reply:
I will update the text as in PJ version as it would be clearer. I will keep the link for now as the code to the button is mostly there so if we decide to reverse back to the button everything could be fixed fast.

Reply:
that works.

Reply:
@ducndh on second thought, can you just bring the revision button back? I feel like it has to be very explicit for the user testing. we can discuss later if we want to change it to the link version or not. if we want to change, we can just come back to use code from the history of this pr. thus, just remove the part with the linked revision page for now.